src/types/boolean.md

```diff
@@ -21,9 +21,10 @@ r[type.bool.layout]
 An object with the boolean type has a [size and alignment] of 1 each.
 
 r[type.bool.repr]
-The value false has the bit pattern `0x00` and the value true has the bit pattern
-`0x01`. It is [undefined behavior] for an object with the boolean type to have
-any other bit pattern.
+A `bool` is represented as a single initialized byte with a value of `0x00` corresponding to `false` and a value of `0x01` corresponding to `true`.
+
+> [!NOTE]
+> No other representations are valid for `bool`. Undefined Behaviour occurs when any other byte is read as type `bool`.
```

Comment:
```suggestion
> No other representations are valid for `bool`. Undefined behavior occurs when any other byte is read as type `bool`.
```

Use American spellings as prescribed in the [style guide](https://github.com/rust-lang/reference/blob/master/docs/authoring.md#style).